react/ProductWrapper.js

```diff
@@ -117,12 +118,9 @@ class ProductWrapper extends Component {
       <div className="vtex-product-context-provider">
         <Helmet>
           {titleTag && <title>{titleTag}</title>}
-          {metaTagDescription && (
```

Comment:
Wouldn't be better for you to include the script tag in `Helmet`?

Reply:
When i put in helmet scope the google tool doesnt regonize the metadata.